pkg/provisioner/provisioner.go

```diff
@@ -110,6 +110,11 @@ func (p *Provisioner) Run(env v1alpha1.Environment) error {
 		p.tpl.Reset()
 	}
 
+	// Close the current ssh connection
+	if err := p.Client.Close(); err != nil {
```

Comment:
Question: Where do we actually open the connection? Should we not call close in a `defer` statement there?

Reply:
we have a  function  `connectOrDie` to create the ssh session, and that func doesn't have the close
https://github.com/NVIDIA/holodeck/blob/main/pkg/provisioner/provisioner.go#L230

Reply:
```	if err = p.Run(opts.cfg); err != nil {
		return fmt.Errorf("failed to run provisioner: %v", err)
	}
	defer p.Client.Close()
```?

Reply:
you mean setting the close at `cmd/cli/create/create.go` ? that's also valid